src/Components/test/testassets/BasicTestApp/ReconnectTicker.razor

```diff
@@ -1,7 +1,14 @@
-@page "/ticker"
-@implements IDisposable
+@implements System.IDisposable
 @using System.Threading
 
+<h1>Reconnect component</h1>
```

Comment:
Why move this to the other project?

Reply:
I'm just a little unclear about our desired organization of these components.

Reply:
I would have had to duplicate the infrastructure in the other server project. These tests weren't supposed to be here in the first place. There's a workitem to clean them up. And @SteveSandersonMS will be incredibly happy about it.

Reply:
👍 

In the past Ryan has expressed concern that `BasicTestApp` will get too big or unfocused. I agree there is need to keep it organized, but until there's a clear reason to have multiple test projects for different kinds of tests, I prefer the one rather than a completely arbitrary division where you can't guess which project a given bit of test infrastructure will be in.